### PR TITLE
bug(hamburger) : reset hamburger after navigation plp and other minor…

### PR DESCRIPTION
**Pull request by kibo-susantasamanta**
… fixes

### CODE REVIEW

File: components/KiboHamburgerMenu.vue [hunk 1]
```diff
@@ -38,7 +38,11 @@
             <div class="category-content">
               <SfList>
                 <SfListItem v-for="(category, key) in megaMenuCategories" :key="key">
-                  <SfMenuItem :label="$t(category.content.name)" @click="goNext(category)" />
+                  <SfMenuItem
+                    :label="$t(category.content.name)"
```
Comment (kibo-kevinwatts) [inline components/KiboHamburgerMenu.vue]: remove $t, we won't have translations for this dynamic data here

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]

File: components/KiboHamburgerMenu.vue (hunk 1 above)
Comment (kibo-susantasamanta) [reply components/KiboHamburgerMenu.vue]: Resolved..

**Review by kibo-kevinwatts — APPROVED** [review]

File: components/KiboHamburgerMenu.vue [hunk 2]
```diff
@@ -135,6 +140,15 @@ export default defineComponent({
         return false
       }
     }
+    const resetHamburger = () => {
+      title.value = "Back"
```
Comment (kibo-chandradeeptalaha) [inline components/KiboHamburgerMenu.vue]: use context.root.$t() for 'Back' and 'All Departments'

**Review by kibo-chandradeeptalaha — CHANGES_REQUESTED** [review]
Please take a look at the comment otherwise looks good

**Review by kibo-sushant — APPROVED** [review]

**Pull request merged**